docs/validation-guide.md

```
@@ -130,4 +161,202 @@ To add new validation checks:
+find . -type d -name __pycache__ -exec rm -rf {} +
+find . -type f -name "*.pyc" -delete
+
+# Reset git state
+git clean -fd
+git reset --hard
+```
+
+#### Check CI Status
+```bash
+# Recent runs
+gh run list --limit 5
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)